index.html

```diff
@@ -31,10 +31,12 @@
 <body>
     <div>
         <nav class="nav-links">
-            <ul class="nav-mobile">
-                <li class="logo"><a href="/">Tolu's Logo</a></li>
-                <li class="handburger" ><a href="/"><i class="fas fa-bars hamburger-icon menu"></i></a></li>
-            </ul>
+            <div class="nav-mobile">
+                <a class="logo" href="/">Tolu's Logo</a>
+                <!-- <a class="handburger" href="/"><i class="fas fa-bars hamburger-icon menu"></i></a> -->
+                <button id="menu-button" class="handburger"><i class="fas fa-bars hamburger-icon menu"></i></button>

```

Comment:
- [ ]  OPTIONAL
It will be a nice idea to make a mouse hover over the hamburger menu change the mouse cursor to a pointer, by adding `cursor:pointer` to your CSS on hover state. This will improve the UX

Reply:
Thank you @rloterh . I will implement the hover right away.